Source/Core/Core/ActionReplay.cpp

```diff
@@ -329,8 +329,11 @@ static void VLogInfo(std::string_view format, fmt::format_args args)
     return;
 
   const bool use_internal_log = s_use_internal_log.load(std::memory_order_relaxed);
-  if (Common::Log::MAX_LOGLEVEL < Common::Log::LogLevel::LINFO && !use_internal_log)
-    return;
+  if constexpr (Common::Log::MAX_EFFECTIVE_LOGLEVEL < Common::Log::LogLevel::LINFO)
```

Comment:
Maybe this is out of scope for the PR, but this looks like a bug. The max effective log level is never less than info. It would make more sense for this code to be checking the current log level.